frontend/src/scenes/insights/Histogram/Histogram.scss

```diff
@@ -1,6 +1,6 @@
 .histogram-container {
     display: flex;
-    min-height: 600px;
+    max-height: 500px;
```

Comment:
Not entirely sure about this, technically, dashboard items can have a much larger height.

<img width="662" alt="" src="https://user-images.githubusercontent.com/5864173/126526234-adf95879-ba4b-4484-8f24-9303e703903d.png">

Reply:
got it, I'll scope this constraint to just insights funnels